scss/_body.scss

```diff
@@ -92,6 +92,12 @@
   }
 }
 
+.n-content-table__cell--hide-mobile {
+  @include oGridRespondTo($until: S) {
+    display: none;
+  }
+}
```

Comment:
**suggestion:** As there currently isn't a defined block of `n-content-table` and this will be within `n-content-body`, we should call this class `n-content-body__table-cell--hide-mobile`.